cmake/EthDependencies.cmake

```diff
@@ -34,10 +34,9 @@ set(BOOST_COMPONENTS "filesystem;unit_test_framework;program_options;system")
 if (WIN32)
 	# Boost 1.77 fixes a bug that causes crashes on Windows for some relative paths in --allow-paths.
 	# See https://github.com/boostorg/filesystem/issues/201
-	find_package(Boost 1.77.0 QUIET REQUIRED COMPONENTS ${BOOST_COMPONENTS})
+	find_package(Boost 1.77.0 EXACT QUIET REQUIRED COMPONENTS ${BOOST_COMPONENTS})
 else()
-	# Boost 1.65 is the first to also provide boost::get for rvalue-references (#5787).
-	find_package(Boost 1.65.0 QUIET REQUIRED COMPONENTS ${BOOST_COMPONENTS})
+	find_package(Boost 1.74.0 QUIET REQUIRED COMPONENTS ${BOOST_COMPONENTS})
```

Comment:
Please mention why 1.78+ does not work on Windows and why versions earlier than 1.65 are too old.

Reply:
For Windows - sure. For 1.65 I don't really see how it makes sense to document versions <1.65 if the minimum we require is 1.74? Also, the reason we're supporting 1.74 is because that's the minimum version used by our CI, and we have no intention of supporting boost versions that we don't use.

Reply:
Ah, sorry, I meant 1.74.

> Also, the reason we're supporting 1.74 is because that's the minimum version used by our CI, and we have no intention of supporting boost versions that we don't use.

Sure. Just put this in a comment. The fact that it's completely arbitrary is even more reason to do it. We have this info at hand now but later can be very frustrating dig out the reason and figure out if it's fine to change it or not.